server/src/main/java/com/linecorp/centraldogma/server/internal/admin/service/MetadataService.java

```diff
@@ -359,6 +407,37 @@ public void initialize() {
                                                       .build())));
     }
 
+    /**
+     * Changes a {@link ProjectRole} of a {@link TokenInfo}.
+     */
+    public CompletionStage<ProjectInfo> changeTokenRole(String projectName, Author author,
+                                                        String appId, ProjectRole projectRole) {
+        requireNonNull(projectName, "projectName");
+        requireNonNull(author, "author");
+        requireNonNull(appId, "appId");
+        requireNonNull(projectRole, "projectRole");
+        final String commitSummary = "Change a projectRole of " + appId + " to " + projectRole +
+                                     " from a project " + projectName;
```

Comment:
- a projectRole -> the projectRole
- from a project -> for the project
- Could you wrap the variables with single quotes? e.g. `'" + appId + "'`